monkey/monkey_island/cc/services/config_schema/basic_network.py

```diff
@@ -11,7 +11,7 @@
             "properties": {
                 "info_box": {
                     "title": "",
-                    "type": "object",
+                    "type": "text",
```

Comment:
I think you might be able to remove these too. In `UiSchema.js`, we're giving the form a custom field (`InfoBox`), so I'm guessing it doesn't bother about the type/title. (I know you can remove type. Not sure about title. I may be missing something.)

Reply:
Title and type can be removed and still to work. I am guessing it doesn't bother about them because of the custom field.